### PR TITLE
Feature lu 3001 db export

### PR DESCRIPTION
**Pull request by ranjit-das**
Endpoint dbExport added for exporting takeon database

### CODE REVIEW

File: src/main/java/uk/gov/ons/collection/controller/ContributorController.java [hunk 1]
```diff
@@ -63,5 +63,67 @@ public String filterAndPrepareSearchParameters(Map<String, String> inputParamete
...
+	System.out.println(responseText.toString());
+        return responseText;
+    }
+
+    @Autowired
+    GraphQLService gqlService;
+    @GetMapping(value = "/dbExport", produces = MediaType.APPLICATION_JSON_VALUE)
+
+    public String validationDbExport(){
+
```
Comment (dllmorgan) [inline src/main/java/uk/gov/ons/collection/controller/ContributorController.java]: Unexpected empty line

File: src/main/java/uk/gov/ons/collection/controller/ContributorController.java [hunk 2]
```diff
@@ -63,5 +63,67 @@ public String filterAndPrepareSearchParameters(Map<String, String> inputParamete
...
+        }
+        catch(Exception e){
+            responseText = "{\"error\":\"Invalid response from graphQL\"}";
+        }
+	System.out.println(responseText.toString());
+        return responseText;
+    }
+
+    @Autowired
+    GraphQLService gqlService;
```
Comment (dllmorgan) [inline src/main/java/uk/gov/ons/collection/controller/ContributorController.java]: I think we can use the same service that has already been declared (qlservice)

File: src/main/java/uk/gov/ons/collection/controller/qlQueryBuilder.java [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+            return "";
+        }        
+        StringBuilder conditions = new StringBuilder();
+        conditions.append("(condition: { ");
+        parameters.forEach((key,value) -> conditions.append(key + ": \\\"" + value + "\\\" "));
+        conditions.append("})");
+        return conditions.toString();
+    }
+
+    public String exportDB(){
```
[the remark this answers is not in the document]
Comment (ranjit-das) [reply src/main/java/uk/gov/ons/collection/controller/qlQueryBuilder.java]: Done

File: graphql.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
```
[the remark this answers is not in the document]
Comment (ranjit-das) [reply graphql.yml]: It is used creating graphql layer in EKS cluster.

File: src/main/java/uk/gov/ons/collection/controller/ContributorController.java (hunk 1 above)
Comment (ranjit-das) [reply src/main/java/uk/gov/ons/collection/controller/ContributorController.java]: Removed empty line.

File: src/main/java/uk/gov/ons/collection/controller/ContributorController.java (hunk 2 above)
Comment (ranjit-das) [reply src/main/java/uk/gov/ons/collection/controller/ContributorController.java]: Same service used now.

**Pull request merged**